examples/npt_equilibration.py

```diff
@@ -0,0 +1,128 @@
+# Run a simulation at constant temperature and pressure, at a variety of values of lambda
+
+import numpy as np
+from simtk import unit
+
+from testsystems.relative import hif2a_ligand_pair
+
+from md.builders import build_water_system
+from md.minimizer import minimize_host_4d
+
+from md.ensembles import PotentialEnergyModel, NPTEnsemble
+
+from md.thermostat.utils import sample_velocities
+
+from md.barostat.utils import get_group_indices
+from md.barostat.moves import MonteCarloBarostat
+
+from md.thermostat.moves import UnadjustedLangevinMove
+
+from md.states import CoordsVelBox
+from md.utils import simulate_npt_traj
+
+from fe.free_energy import AbsoluteFreeEnergy, construct_lambda_schedule
+
+from timemachine.lib import LangevinIntegrator
+from functools import partial
+
+import matplotlib.pyplot as plt
+
+# simulation parameters
+n_lambdas = 40
+initial_waterbox_width = 3.0 * unit.nanometer
+timestep = 1.5 * unit.femtosecond
+collision_rate = 1.0 / unit.picosecond
+n_moves = 2000
+barostat_interval = 5
+seed = 2021
+
+# thermodynamic parameters
+temperature = 300 * unit.kelvin
+pressure = 1.0 * unit.atmosphere
```

Comment:
Should we be consistent with OpenMM for sanity purposes? IIRC the default was 1 bar (which is very close but not exactly to 1 atm)

Reply:
Updated the examples from atm to bar in https://github.com/proteneer/timemachine/pull/433/commits/0f483831b931ee8ed3844391a9aa5add835290ca. (Note that we don't set a default temperature or pressure in the `barostat` or `ensembles` code itself, and allow user inputs in any simtk pressure unit.)